src/plugins/workspace/public/workspace_client.ts

```diff
@@ -150,6 +156,7 @@ export class WorkspaceClient implements IWorkspaceClient {
         result: null,
       };
     } else {
+      workspaceError$.next(workspaceResp.error!);
```

Comment:
Can we just call `this.workspaces.workspaceError$.next(workspaceResp.error!)`? We do not need to change the public method type by doing so.